### DISCUSSION TITLE
How to disable GitHub pull request merge button when CI job got failed?

### DISCUSSION THREAD

**Discussion by ElangoRajendran** [cake-build/cake#3351] [Q&A] [answered]
How to disable GitHub pull request merge button when the CI job got failed?
Merge button got enable only CI job got success otherwise it should be disabled.

The user could not merge the pull request changes when the CI job got failed. The user can merge only CI job got success.

Are there any possibilities to do this?

Could you suggest a solution for this?

**Comment by augustoproiete** [comment, marked as answer]
@ElangoRajendran You're looking for [branch protection rules](https://docs.github.com/en/github/administering-a-repository/managing-a-branch-protection-rule) in GitHub repositories.

![image](https://user-images.githubusercontent.com/177608/117827527-7200a880-b247-11eb-97fd-40b59dc93d65.png)

**Reply by ElangoRajendran** [reply]
Hi @augustoproiete,

Thanks for the update, it’s working fine.

After enabling the Require status checks before merging, I have added value as build on Status checks that are required column, it is working fine.

Regards,
Elango R.
